mmdet/models/losses/gfocal_loss.py

```diff
@@ -186,3 +216,62 @@ def forward(self,
         loss_cls = self.loss_weight * distribution_focal_loss(
             pred, target, weight, reduction=reduction, avg_factor=avg_factor)
         return loss_cls
+
+
+@LOSSES.register_module()
+class LocalizationDistillationLoss(nn.Module):
```

Comment:
We may create a new file, e.g. `kd_loss.py` and put this Module to that file since `LocalizationDistillationLoss` seems to be independent.